meta/v1_scope.md

```diff
@@ -0,0 +1,14 @@
+# Initial-Version Scope
+
+The goal and scope of this initial version is to produce a usable rule book for *International Bible Quizzing* (IBQ) and district level quizzing that is *functionality equivalent* (FE) to the existing or current rule book. What this means is that if someone were to read the completed product of this project's initial version, they should render a quiz and meet that is indistinguishable from quizzes and meets at the IBQ level.
```

Comment:
I kinda want "IBQ level" to be rephrased or even defined....we're kinda shooting for "what most everyone, but specifically IBQ, does" idk.

Reply:
I'll see if I can word-smith something.